### PR TITLE
Output Retrieval URL in json response from content/add

### PR DESCRIPTION
**Pull request by snissn**
```
curl -s -X POST http://localhost:3004/content/add -H "Authorization: Bearer key" -H "Content-Type: multipart/form-data" -F "data=@random.randomfile"| jq
{
  "cid": "bafkreig6pkmkrpjl4dxvs73szidywfnleq5ky5pumstouijtr6joispmma",
  "RetrievalURL": "https://dweb.link/ipfs/bafkreig6pkmkrpjl4dxvs73szidywfnleq5ky5pumstouijtr6joispmma",
  "estuaryId": 14,
  "providers": [
    "/ip4/10.128.0.2/tcp/6744/p2p/12D3KooWESGCFHLoXVVEh8xtndgGt3JtkaAG4s6NxEYb2Z4UoK92",
    "/ip4/127.0.0.1/tcp/6744/p2p/12D3KooWESGCFHLoXVVEh8xtndgGt3JtkaAG4s6NxEYb2Z4UoK92",
    "/ip4/35.192.108.226/tcp/6744/p2p/12D3KooWESGCFHLoXVVEh8xtndgGt3JtkaAG4s6NxEYb2Z4UoK92"
  ]
}
```
add RetrievalURL output to the json returned the /content/add endpoint. The rationale is that new users may not know how to retrieve their content, and also other APIs consuming from this will want to know how to retrieve their data. TODO would be to ensure that createRetrievalURL only exists once in a util function, currently its copied into two functions for developer convenience.

### CODE REVIEW

File: cmd/estuary-shuttle/main.go [hunk 1]
```diff
@@ -1381,6 +1383,9 @@ func (s *Shuttle) loadCar(ctx context.Context, bs blockstore.Blockstore, r io.Re
 	return car.LoadCar(ctx, bs, r)
 }
 
+func createRetrievalURL(cid string) string {
```
Comment (en0ma) [inline cmd/estuary-shuttle/main.go:1386]: move this method into the `util.content` package and use in main and shuttle main

File: handlers.go [hunk 2]
```diff
@@ -341,6 +341,10 @@ func serveProfile(c echo.Context) error {
 	return nil
 }
 
+func createRetrievalURL(cid string) string {
```
Comment (en0ma) [inline handlers.go:344]: see comment above

**Comment by en0ma** [comment]
nice addition @snissn - thanks!

**Comment by snissn** [comment]
@en0ma the requested changes were made two hours ago in 

https://github.com/application-research/estuary/pull/364/commits/a31359c2605a3054391584c0dd865eedeb7618f3
https://github.com/application-research/estuary/pull/364/commits/c447d12241049a7dba65309c404aea2ee048484a

**Comment by en0ma** [comment]
> @en0ma the requested changes were made two hours ago in
> 
> [a31359c](https://github.com/application-research/estuary/commit/a31359c2605a3054391584c0dd865eedeb7618f3) [c447d12](https://github.com/application-research/estuary/commit/c447d12241049a7dba65309c404aea2ee048484a)

@snissn your build is failing. I will push and fix and approve it

File: util/content.go [hunk 3]
```diff
@@ -122,3 +124,7 @@ func EnsurePathIsLinked(dirs []string, rootNode *merkledag.ProtoNode, ds format.
 	}
 	return lookupNode, nil
 }
+
+func createRetrievalURL(cid string) string {
```
Comment (en0ma) [inline util/content.go:128]: Please export this method

**Comment by en0ma** [comment]
@snissn I am closing this. It has been addressed in https://github.com/application-research/estuary/pull/376. Thank you very much

**Comment by snissn** [comment]
amazing! thanks for including it!

**Pull request closed without merging**